cosmic-core/utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -508,6 +513,23 @@ public static boolean validIpRange(final String startIP, final String endIP) {
         return startIPLong <= endIPLong;
     }
 
+    public static boolean validIpRange(final String ipRange) {
+        if(StringUtils.isEmpty(ipRange)) {
+            return false;
+        }
+
+        final String[] ipAddresses = ipRange.split("-");
+
+        if(!(ipAddresses.length == 2)) {
```

Comment:
Lines 523 to 530 can be simplified in a single expression: `return (ipAddresses.length == 2) && isValidIp(ipAddresses[0]) && isValidIp(ipAddresses[1]) && validIpRange(ipAddresses[0], ipAddresses[1]);`